diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f141ac..76b7ea3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -383,7 +383,6 @@ set(KTX_MAIN_SRC
     external/basisu/transcoder/basisu_transcoder.cpp
     external/basisu/transcoder/basisu_transcoder.h
     external/basisu/transcoder/basisu.h
-    external/basisu/zstd/zstd.c
     lib/checkheader.c
     external/dfdutils/createdfd.c
     external/dfdutils/colourspaces.c
@@ -431,6 +430,8 @@ if (KTX_FEATURE_ETC_UNPACK)
 	)
 endif()
 
+find_package(zstd REQUIRED CONFIG)
+
 set(BASISU_ENCODER_CXX_SRC
     external/basisu/encoder/basisu_backend.cpp
     external/basisu/encoder/basisu_backend.h
@@ -587,9 +588,6 @@ macro(common_libktx_settings target enable_write library_type)
         ${KTX_BASISU_INCLUDE_DIRS}
         external
 
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/external/basisu/zstd>
-        $<INSTALL_INTERFACE:external/basisu/zstd>
-
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/utils>
         $<INSTALL_INTERFACE:utils>
     )
@@ -937,10 +935,6 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
             external/basisu/transcoder/basisu_transcoder.cpp
             PROPERTIES COMPILE_OPTIONS "-Wno-sign-compare;-Wno-unused-function;-Wno-unused-variable"
         )
-        set_source_files_properties(
-            external/basisu/zstd/zstd.c
-            PROPERTIES COMPILE_OPTIONS "-Wno-unused-function"
-        )
     endif()
 else()
     message(FATAL_ERROR "${CMAKE_CXX_COMPILER_ID} not yet supported.")
@@ -954,7 +948,6 @@ get_source_file_property(transcoder_options
     external/basisu/transcoder/basisu_transcoder.cpp
     COMPILE_OPTIONS
 )
-get_source_file_property(zstd_options external/basisu/zstd/zstd.c COMPILE_OPTIONS)
 
 if(EMSCRIPTEN)
     set(
