diff --git a/external/basisu/encoder/cppspmd_sse.h b/external/basisu/encoder/cppspmd_sse.h
index 030824b..82b1059 100644
--- a/external/basisu/encoder/cppspmd_sse.h
+++ b/external/basisu/encoder/cppspmd_sse.h
@@ -490,7 +490,7 @@ struct spmd_kernel
 		return dst;
 	}
 
-	CPPSPMD_FORCE_INLINE vfloat& store(vfloat&& dst, const vfloat& src)
+	CPPSPMD_FORCE_INLINE vfloat store(vfloat&& dst, const vfloat& src)
 	{
 		dst.m_value = blendv_mask_ps(dst.m_value, src.m_value, _mm_castsi128_ps(m_exec.m_mask));
 		return dst;
@@ -502,7 +502,7 @@ struct spmd_kernel
 		return dst;
 	}
 
-	CPPSPMD_FORCE_INLINE vfloat& store_all(vfloat&& dst, const vfloat& src)
+	CPPSPMD_FORCE_INLINE vfloat store_all(vfloat&& dst, const vfloat& src)
 	{
 		dst.m_value = src.m_value;
 		return dst;
@@ -2102,4 +2102,3 @@ CPPSPMD_FORCE_INLINE const float_vref& spmd_kernel::store_all(const float_vref&&
 #include "cppspmd_math.h"
 
 } // namespace cppspmd_sse41
-
